request_token/models.py

```diff
@@ -328,6 +322,11 @@ def rmg(key, default=None):
         self.save()
         return log
 
+    def invalidate(self):
```

Comment:
Invalidation of the token could take many forms - expiry, max uses, not before, ... - this seems like the simplest, and has the added advantage of timestamping when the invalidation occurred.

Reply:
Another possible name rather than `invalidate` would be `expire`. Then in the function it would be `token.expire()` and you'd know exactly what it was doing without having to check how it did it.